README.md

```diff
@@ -52,17 +45,15 @@ Global Redirect modules, which allow you to set forwarding either per item or
 across the site to your aliased URLs.
 
 URLs (not) Getting Replaced With Aliases:
-Please bear in mind that only URLs passed through Drupal's l() or url()
-functions will be replaced with their aliases during page output. If a module
-or your template contains hardcoded links, such as 'href="node/$node->nid"'
-those won't get replaced with their corresponding aliases. Use the
-Drupal API instead:
-
-* 'href="'. url("node/$node->nid") .'"' or
-* l("Your link title", "node/$node->nid")
-
-See http://api.drupal.org/api/HEAD/function/url and
-http://api.drupal.org/api/HEAD/function/l for more information.
+Please bear in mind that only URLs passed through Drupal's Link::fromTextAndUrl
```

Comment:
I could not find any documentation that centralizes this, but instead an issue where core devs are debating that there are way too many ways to generate a link :-D: https://www.drupal.org/node/2491981

I will follow your advice and change the wording.